index.html

```diff
@@ -68,6 +68,9 @@ <h1 class="career">I’m a Software<br>Developer</h1>
 
     <!--work section-->
     <section id="my-work" class="my-work">
+        <div class="popup-container">
+            
+        </div>
```

Comment:
Your pop-up feature is only working for the first project because you didn't follow the project requirement.

According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20use%20a%20JavaScript%20array%20to%20store%20all%20of%20the%20information%20for%20all%20projects.), you are supposed to use a JavaScript array to store all of the information for all projects, however, you hardcoded the projects in your `index.html`.